packages/php/remote-specs-validation/bin/build

```
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+/* eslint-disable @typescript-eslint/no-var-requires */
+const JsonRefs = require( 'json-refs' );
+const fs = require( 'fs' );
+const yaml = require( 'js-yaml' );
+const commander = require( 'commander' );
+const version = require( '../package.json' ).version;
+const path = require( 'path' );
+
+let indexJsonValue;
+
+function bundle( indexJson ) {
+	JsonRefs.clearCache();
+	const root = yaml.load( fs.readFileSync( indexJson, 'utf8' ) );
+	const options = {
+		filter: [ 'relative', 'remote' ],
+		resolveCirculars: true,
+		location: indexJson,
+		loaderOptions: {
+			processContent( res, callback ) {
+				callback( undefined, yaml.load( res.text ) );
+			},
+		},
+	};
+
+	function dictToString( dict ) {
+		const res = [];
+		for ( const [ k, v ] of Object.entries( dict ) ) {
+			res.push( `${ k }: ${ v }` );
+		}
+		return res.join( '\n' );
+	}
+
+	return JsonRefs.resolveRefs( root, options ).then(
+		function ( results ) {
+			const resErrors = {};
+			for ( const [ k, v ] of Object.entries( results.refs ) ) {
+				if (
+					'missing' in v &&
+					v.missing === true &&
+					( v.type == 'relative' || v.type === 'remote' )
```

Should we use a strict comparison here?
```suggestion
					( v.type === 'relative' || v.type === 'remote' )
```